app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
```

Comment:
`SampleApp`
Em tìm hiểu cho chị: `I18n in rails` và sử dụng i18n vào tất cả các text ở html file nhé